.github/workflows/push.yml

```diff
@@ -15,6 +15,8 @@ jobs:
   rex-lint:
     name: REX Linting
     runs-on: ubuntu-latest
+    if: github.event.pull_request.draft == false
```

Comment:
Ich glaube das Linting würde ich eher aktiv lassen. Ist ja nicht ganz verkehrt, frühzeitig zu erfahren, dass man zum Beispiel PHP-Sprachkonstrukte verwendet hat, die es in unserer niedrigsten PHP-Version noch nicht gibt, oder?

Aber ist nur so ein Gedanke, bin auch mit einverstanden, wenn wir es doch deaktivieren für Drafts.

Reply:
Da wir solche probleme bisher nicht/nur sehr selten haben, finde ich reicht es im non-draft state.

Die meisten nutzen ohnehin gar keine draft PRs :)

Reply:
Falls wir andere Erfahrungen machen können wir es ja noch ändern :)